libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -611,19 +617,36 @@ bool AP_Follow::handle_global_position_int_message(const mavlink_message_t &msg)
         return false;
     }
 
+    // apply jitter-corrected timestamp to this update
```

Comment:
This needs to be moved down where it was set before. It can be updated if the message is not valid.

Reply:
Are you sure? If you do this, a lot of internal values will be updated with new calculations based on a message that is going to be discarded, I'm not sure I understand why that is ok.